tests/utils/helpers.py

```diff
@@ -196,3 +196,35 @@ def _flatten(item):
 
     _flatten(data)
     return result
+
+
+def flatten_tx_accesslist(access_list):
+    """
+    Transform the access list from the transaction dict into a flattened list of
+    [address, storage_keys, ...].
+    """
+    result = []
+    for item in access_list:
+        result.append(int(item["address"], 16))
+        for key in item["storageKeys"]:
+            value = int(key, 16)
+            value_low = value & (2**128 - 1)
+            value_high = value >> 128
+            result.append(value_low)
+            result.append(value_high)
+    return result
+
+
+def serialize_accesslist(access_list):
```

Comment:
it's deserialize then? because serialization = cairo => json/dict, and deserialization is json/dict => cairo

Reply:
actually, it wasn't used.